sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -952,9 +942,35 @@ export class MessageReceiver extends LinkEntity {
           connectionHost: this._context.namespace.config.host,
           delayInSeconds: 15
         };
-        if (!this.wasCloseInitiated) {
-          await retry<void>(config);
-        }
+        await retry<void>(config);
+      } else if (causedByDisconnect) {
+        const state: any = {
+          wasCloseInitiated: this.wasCloseInitiated,
```

Comment:
what is the point of logging `this.wasCloseInitiated`? If that is true, then we would have done an early exit right?

Reply:
Yes, this is no longer valuable to know.